app/assets/stylesheets/page_specific/_error.scss

```diff
@@ -0,0 +1,8 @@
+.center {
```

Comment:
Minor point: don't think we would want to add a specific style for centred content instead of adding the text-align: center style to the .error-content class.